guides/ip-blacklist.md

```diff
@@ -4,6 +4,8 @@ When used in client-side Javascript, Rollbar access tokens are visible to end us
 
 Each Rollbar project can be configured with a blacklist of IP addresses from whom we will not accept any data.
 
-To configure the blacklist for your project, go to **Project Settings --> IP Blacklist**, and enter one or more IP addresses, separated by newlines or commas.
+To configure the blacklist for your project, go to **Project Settings --> IP Blacklist**, and enter one or more valid IPv4/IPv6 addresses or subnets, separated by newlines or commas.  Examples of valid entries include:
```

Comment:
how about: "one more more valid IPv4/IPv6 addresses or subnets (in CIDR notation)"